ginga/AstroImage.py

```diff
@@ -815,10 +815,10 @@ def info_xy(self, data_x, data_y, settings):
                 # log traceback, if possible
                 (type_, value_, tb) = sys.exc_info()
                 tb_str = "".join(traceback.format_tb(tb))
-                self.logger.error("Traceback:\n%s" % (tb_str))
+                self.logger.warning("Traceback:\n%s" % (tb_str))
             except Exception:
                 tb_str = "Traceback information unavailable."
-                self.logger.error(tb_str)
+                self.logger.warning(tb_str)
```

Comment:
@pllim, any particular reason for changing the logging of this traceback to a warning?

Reply:
Yeah, this was a leftover from my attempt to silence the bad WCS calculations, as GWCS returns `nan` for RA and Dec for pixels outside of image bounds. It didn't really work as I thought it would anyway, so I will go ahead and revert this change here. Thanks for catching it.